### ISSUE TITLE
Bottom padding missing in Firefox

### ISSUE THREAD

**Issue by DennieMello** [Andarist/react-textarea-autosize#316]
The bottom padding is inconsistent/missing in Firefox.  Everything is fine in Сhrome. May be related to spec issue w3c/csswg-drafts#129.
![Снимок экрана 2021-03-25 в 16 53 00](https://user-images.githubusercontent.com/39631512/112484055-a8af5c00-8d8a-11eb-97df-7e242796b086.jpg)

**Comment by Andarist** [comment]
If you can propose how to fix this and provide a PR for - I could consider a proposed change. Otherwise, I won't be acting on this.

**Comment by tskytt** [comment]
The problem is gone in the nightly build after the release of Firefox 87.0, so it was a bug in Firefox, which fortunately has been fixed now.

**Comment by anttibull** [comment]
I have version 87.0 and the problem is still there

**Comment by tskytt** [comment]
Yes, the problem is in 87.0, but has disappeared in the build after 87.0. That was what I was trying to say 😉. So I guess and hope we won't see the problem when version 88 is released.

**Comment by desmondmc** [comment]
Still an issue in Firefox 88 😿

**Comment by cusxio** [comment]
Seems like this is fixed in Firefox 89.
